src/ui/reusable/RealmLoadingComponent/index.tsx

```diff
@@ -169,6 +170,7 @@ export abstract class RealmLoadingComponent<
                 }
               },
             },
+            */
```

Comment:
Disabling the initial subscription, since this could potentially result in a huge download.